airflow/providers/telegram/provider.yaml

```diff
@@ -39,7 +40,7 @@ dependencies:
   # The telegram bot 20.0.0 is not yet compatible with our provider as documented in
   # https://github.com/apache/airflow/issues/28670. This limit should be removed (and likely replaced
   # with >=20.0.0) once the issue is fixed.
-  - python-telegram-bot>=13.0,<20.0.0
+  - python-telegram-bot>=20.0.0
```

Comment:
Can you please also remove the comments in the linea above? They are no longer relevant

Reply:
Done